integration_tests/__init__.py

```diff
@@ -78,13 +78,17 @@ def get_output_ignoring_non_zero_exit(self, binary, cwd):
             output = exception.output
         return output
 
-    def login(self, email='u1test+snapcraft@canonical.com',
+    def login(self, email=None,
               password=None, expect_success=True):
-        password = password or os.getenv('TEST_USER_PASSWORD', None)
+        email = email or os.getenv(
+            'TEST_USER_EMAIL', 'u1test+snapcraft@canonical.com')
+        password = password or os.getenv(
+            'TEST_USER_PASSWORD', 'test correct password')
```

Comment:
shouldn't `test correct password` be an empty string? Or is the `self.skipTest` call not needed anymore?